wai-aria/role/invalid-roles.html

```diff
@@ -11,26 +11,109 @@
 </head>
 <body>
 
-<p>Verifies Invalid Roles from <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a></p>
+<p>Tests invalid ARIA roles from <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a></p>
 
-<nav role="foo" data-testname="invalid role name foo" data-expectedrole="navigation" class="ex">x</nav>
-<nav role="foo bar" data-testname="multiple invalid role names" data-expectedrole="navigation" class="ex">x</nav>
+<!-- Tests fallback for <nav> when 1/2/3 invalid roles are supplied -->
+<nav role="foo" data-testname="nav with invalid role name foo" data-expectedrole="navigation" class="ex">x</nav>
+<nav role="foo bar" data-testname="nav with two invalid role names" data-expectedrole="navigation" class="ex">x</nav>
+<nav role="foo bar baz" data-testname="nav with three invalid role names" data-expectedrole="navigation" class="ex">x</nav>
 
-<!-- todo: additional invalid roles:
+<!-- Tests fallback for <button> when 1/2/3 invalid roles are supplied -->
+<button role="foo" data-testname="button with invalid role name foo" data-expectedrole="button" class="ex">x</button>
+<button role="foo bar" data-testname="button with two invalid role names" data-expectedrole="button" class="ex">x</button>
+<button role="foo bar baz" data-testname="button with three invalid role names" data-expectedrole="button" class="ex">x</button>
```

Comment:
is the `nav` vs `button` variant here testing container vs leaf node?

Reply:
@cookiecrook Yes, and also common non-interactive/interactive elements.

Reply:
okay